ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WSQueryEndpoint.java

```diff
@@ -213,10 +213,10 @@ public void onOpen(final Session session, final EndpointConfig unused) {
 
       serviceContext = createServiceContext(session.getUserPrincipal());
 
-      authorizationValidator.checkAuthorization(
+      authorizationValidator.ifPresent(validator -> validator.checkAuthorization(
```

Comment:
This websocket endpoint handles pull queries too, so should have consistent handling of auth checks as the rest endpoint.

Reply:
Let’s keep parity for now.

Reply:
This endpoint is used by the KSQL UI. Better to support it now than forget it later.